### PR TITLE
add TRACER_DB_TYPE="clickhouse" to env

### PR DESCRIPTION
**Pull request by iguberman**

### CODE REVIEW

**Comment by neonlabstech** [comment]
<details><summary>Solana Requests Statistics</summary>



|                                   |   count |   min_time |   max_time |   average_time |   median_time |
|:----------------------------------|--------:|-----------:|-----------:|---------------:|--------------:|
| getSlot                           |   63495 |          0 |      0.244 |    0.000871706 |        0      |
| getAccountInfo                    |   58461 |          0 |      0.1   |    0.00125694  |        0      |
| getBlocks                         |   25441 |          0 |      0.072 |    0.000596871 |        0      |
| getBlock                          |   16565 |          0 |      0.063 |    0.00126091  |        0.001  |
| getMultipleAccounts               |   14093 |          0 |      0.13  |    0.00193273  |        0.001  |
| getTransaction                    |    7983 |          0 |      0.256 |    0.0100665   |        0.004  |
| getBlockTime                      |    6296 |          0 |      0.056 |    0.000425985 |        0      |
| sendTransaction                   |    5179 |          0 |      0.142 |    0.0079386   |        0.003  |
| getLatestBlockhash                |    4981 |          0 |      0.255 |    0.00271431  |        0.001  |
| getBalance                        |    1823 |          0 |      0.034 |    0.00138398  |        0      |
| getSignatureStatuses              |    1122 |          0 |      0.033 |    0.00100178  |        0      |
| getVersion                        |     585 |          0 |      0.03  |    0.00116239  |        0      |
| isBlockhashValid                  |     543 |          0 |      0.005 |    0.000292818 |        0      |
| getSignaturesForAddress           |      61 |          0 |      0.012 |    0.00145902  |        0.001  |
| simulateTransaction               |      38 |          0 |      0.001 |    0.000447368 |        0      |
| requestAirdrop                    |      32 |          0 |      0.001 |    0.0004375   |        0      |
| getHealth                         |       3 |          0 |      0     |    0           |        0      |
| getTokenAccountBalance            |       3 |          0 |      0.001 |    0.000333333 |        0      |
| getFirstAvailableBlock            |       2 |          0 |      0.001 |    0.0005      |        0.0005 |
| getMinimumBalanceForRentExemption |       1 |          0 |      0     |    0           |        0      |
| getClusterNodes                   |       1 |          0 |      0     |    0           |        0      |

</details>

File: common/neon_rpc/server.py [hunk 1]
```diff
@@ -67,6 +67,7 @@ def _create_env(self) -> dict[str, Any]:
             COMMITMENT="recent",
             EVM_LOADER=str(NeonProg.ID),
             NEON_DB_CLICKHOUSE_URLS=";".join(self._cfg.ch_dsn_list),
+            TRACER_DB_TYPE="clickhouse",
```
Comment (s-medvedev) [inline common/neon_rpc/server.py]: Set up `TRACER_DB_TYPE` to `clickhouse` or `none` depends on `self._cfg.ch_dsn_list` value.

**Comment by neonlabstech** [comment]
<details><summary>Solana Requests Statistics</summary>



|                                   |   count |   min_time |   max_time |   average_time |   median_time |
|:----------------------------------|--------:|-----------:|-----------:|---------------:|--------------:|
| getAccountInfo                    |   59158 |      0     |      0.097 |    0.00121556  |        0      |
| getSlot                           |   58299 |      0     |      0.159 |    0.000772397 |        0      |
| getBlocks                         |   23113 |      0     |      0.038 |    0.000595466 |        0      |
| getBlock                          |   14948 |      0     |      0.162 |    0.0012993   |        0.001  |
| getMultipleAccounts               |   13744 |      0     |      0.225 |    0.00164268  |        0.001  |
| getLatestBlockhash                |   10992 |      0     |      0.333 |    0.00129521  |        0      |
| getTransaction                    |    8154 |      0     |      0.094 |    0.00806819  |        0.003  |
| getBlockTime                      |    6265 |      0     |      0.028 |    0.000377654 |        0      |
| sendTransaction                   |    5310 |      0     |      0.117 |    0.00665235  |        0.003  |
| getBalance                        |    1855 |      0     |      0.064 |    0.00122911  |        0      |
| getSignatureStatuses              |    1187 |      0     |      0.04  |    0.0010059   |        0      |
| getVersion                        |     626 |      0     |      0.031 |    0.00114856  |        0      |
| isBlockhashValid                  |     575 |      0     |      0.012 |    0.000330435 |        0      |
| getSignaturesForAddress           |      63 |      0     |      0.005 |    0.00142857  |        0.001  |
| simulateTransaction               |      38 |      0     |      0.001 |    0.000368421 |        0      |
| requestAirdrop                    |      32 |      0     |      0.001 |    0.0005625   |        0.001  |
| getTokenAccountBalance            |       7 |      0     |      0.022 |    0.00371429  |        0.001  |
| getHealth                         |       3 |      0     |      0.001 |    0.000666667 |        0.001  |
| getFirstAvailableBlock            |       2 |      0     |      0.001 |    0.0005      |        0.0005 |
| getMinimumBalanceForRentExemption |       1 |      0.001 |      0.001 |    0.001       |        0.001  |
| getClusterNodes                   |       1 |      0.001 |      0.001 |    0.001       |        0.001  |

</details>

**Comment by neonlabstech** [comment]
<details><summary>Solana Requests Statistics</summary>



|                                   |   count |   min_time |   max_time |   average_time |   median_time |
|:----------------------------------|--------:|-----------:|-----------:|---------------:|--------------:|
| getAccountInfo                    |   59441 |      0     |      0.121 |    0.00164375  |        0      |
| getSlot                           |   58848 |      0     |      0.081 |    0.00094739  |        0      |
| getBlocks                         |   23740 |      0     |      0.054 |    0.00064861  |        0.001  |
| getBlock                          |   15318 |      0     |      0.052 |    0.00147447  |        0.001  |
| getMultipleAccounts               |   14024 |      0     |      0.211 |    0.00213926  |        0.001  |
| getLatestBlockhash                |   11124 |      0     |      0.19  |    0.00171359  |        0.001  |
| getTransaction                    |    8147 |      0     |      0.099 |    0.00959175  |        0.004  |
| getBlockTime                      |    6368 |      0     |      0.053 |    0.000499215 |        0      |
| sendTransaction                   |    5321 |      0     |      0.136 |    0.00810712  |        0.004  |
| getBalance                        |    1857 |      0     |      0.077 |    0.00173829  |        0.001  |
| getSignatureStatuses              |    1190 |      0     |      0.119 |    0.00168403  |        0.001  |
| getVersion                        |     637 |      0     |      0.074 |    0.00171743  |        0      |
| isBlockhashValid                  |     577 |      0     |      0.026 |    0.000585789 |        0      |
| getSignaturesForAddress           |      62 |      0     |      0.016 |    0.00282258  |        0.001  |
| simulateTransaction               |      38 |      0     |      0.001 |    0.000394737 |        0      |
| requestAirdrop                    |      32 |      0     |      0.001 |    0.0008125   |        0.001  |
| getTokenAccountBalance            |      12 |      0     |      0.048 |    0.00475     |        0.001  |
| getHealth                         |       3 |      0     |      0.001 |    0.000333333 |        0      |
| getFirstAvailableBlock            |       2 |      0     |      0.001 |    0.0005      |        0.0005 |
| getMinimumBalanceForRentExemption |       1 |      0.001 |      0.001 |    0.001       |        0.001  |
| getClusterNodes                   |       1 |      0     |      0     |    0           |        0      |

</details>

**Comment by neonlabstech** [comment]
<details><summary>Solana Requests Statistics</summary>



|                                   |   count |   min_time |   max_time |   average_time |   median_time |
|:----------------------------------|--------:|-----------:|-----------:|---------------:|--------------:|
| getAccountInfo                    |   59313 |      0     |      0.178 |    0.00172873  |         0     |
| getSlot                           |   58459 |      0     |      0.152 |    0.0010454   |         0     |
| getBlocks                         |   23215 |      0     |      0.174 |    0.000704114 |         0.001 |
| getBlock                          |   14982 |      0     |      0.034 |    0.0015303   |         0.001 |
| getMultipleAccounts               |   14059 |      0     |      0.311 |    0.00233096  |         0.001 |
| getLatestBlockhash                |   11007 |      0     |      0.142 |    0.00194576  |         0.001 |
| getTransaction                    |    8164 |      0     |      0.342 |    0.0129708   |         0.005 |
| getBlockTime                      |    6405 |      0     |      0.09  |    0.000561436 |         0     |
| sendTransaction                   |    5299 |      0     |      0.177 |    0.00945273  |         0.004 |
| getBalance                        |    1852 |      0     |      0.062 |    0.00187311  |         0.001 |
| getSignatureStatuses              |    1180 |      0     |      0.114 |    0.00136949  |         0.001 |
| getVersion                        |     618 |      0     |      0.099 |    0.00137864  |         0     |
| isBlockhashValid                  |     572 |      0     |      0.048 |    0.000562937 |         0     |
| getSignaturesForAddress           |      63 |      0.001 |      0.003 |    0.0015873   |         0.001 |
| simulateTransaction               |      38 |      0     |      0.001 |    0.000315789 |         0     |
| requestAirdrop                    |      32 |      0     |      0.001 |    0.0006875   |         0.001 |
| getTokenAccountBalance            |       5 |      0     |      0.001 |    0.0004      |         0     |
| getHealth                         |       3 |      0     |      0.002 |    0.001       |         0.001 |
| getFirstAvailableBlock            |       2 |      0.001 |      0.001 |    0.001       |         0.001 |
| getMinimumBalanceForRentExemption |       1 |      0.001 |      0.001 |    0.001       |         0.001 |
| getClusterNodes                   |       1 |      0     |      0     |    0           |         0     |

</details>

**Comment by neonlabstech** [comment]
<details><summary>Solana Requests Statistics</summary>



|                                   |   count |   min_time |   max_time |   average_time |   median_time |
|:----------------------------------|--------:|-----------:|-----------:|---------------:|--------------:|
| getSlot                           |   63904 |      0     |      0.143 |    0.00092977  |         0     |
| getAccountInfo                    |   59393 |      0     |      0.197 |    0.00167439  |         0     |
| getBlocks                         |   26502 |      0     |      0.159 |    0.000657573 |         0.001 |
| getBlock                          |   17199 |      0     |      0.037 |    0.0015259   |         0.001 |
| getMultipleAccounts               |   14744 |      0     |      0.129 |    0.00204849  |         0.001 |
| getLatestBlockhash                |   11827 |      0     |      0.196 |    0.00159964  |         0.001 |
| getTransaction                    |    8134 |      0     |      0.195 |    0.0114116   |         0.004 |
| getBlockTime                      |    6436 |      0     |      0.037 |    0.000590273 |         0     |
| sendTransaction                   |    5279 |      0     |      0.192 |    0.00797424  |         0.004 |
| getBalance                        |    1848 |      0     |      0.085 |    0.00163853  |         0.001 |
| getSignatureStatuses              |    1162 |      0     |      0.123 |    0.00127108  |         0     |
| getVersion                        |     605 |      0     |      0.037 |    0.00146116  |         0     |
| isBlockhashValid                  |     563 |      0     |      0.029 |    0.000477798 |         0     |
| getSignaturesForAddress           |      61 |      0     |      0.004 |    0.00177049  |         0.002 |
| simulateTransaction               |      38 |      0     |      0.001 |    0.000421053 |         0     |
| requestAirdrop                    |      32 |      0     |      0.001 |    0.0006875   |         0.001 |
| getHealth                         |       3 |      0     |      0.001 |    0.000333333 |         0     |
| getTokenAccountBalance            |       3 |      0     |      0.001 |    0.000333333 |         0     |
| getFirstAvailableBlock            |       2 |      0     |      0     |    0           |         0     |
| getMinimumBalanceForRentExemption |       1 |      0     |      0     |    0           |         0     |
| getClusterNodes                   |       1 |      0.002 |      0.002 |    0.002       |         0.002 |

</details>

**Review by s-medvedev — APPROVED** [review]

**Pull request merged**